backends/nats/nats.go

```diff
@@ -0,0 +1,486 @@
+package nats
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"errors"
+	"github.com/PowerDNS/simpleblob"
+	"github.com/go-logr/logr"
+	"github.com/nats-io/nats.go"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
+	// DefaultMaxReconnects - Max Reconnect attempts
+	DefaultMaxReconnects = 5
+	// DefaultReconnectWaitSeconds - Time to wait before attempting reconnect
+	DefaultReconnectWaitSeconds = 1
```

Comment:
Use `time.Duration` instead, like
```suggestion
	DefaultReconnectWait = 1 * time.Second
```